err_test.go

```diff
@@ -343,6 +343,41 @@ func TestWithClues(t *testing.T) {
 	}
 }
 
+func TestValuePriority(t *testing.T) {
+	table := []struct {
+		name   string
+		err    error
+		expect msa
+	}{
+		{
+			name: "lowest data wins",
+			err: func() error {
+				ctx := clues.Add(context.Background(), "in-ctx", 1)
+				err := clues.NewWC(ctx, "err").With("in-err", 1)
+				ctx = clues.Add(ctx, "in-ctx", 2)
+				err = clues.StackWC(ctx, err).With("in-err", 2)
```

Comment:
could you please reorder the context additions? It's a little confusing because it's the opposite of what usually happens in code. Generally context kv-pair sets grow as you go down the call stack and error kv-pair sets grow as you go up the call stack
```suggestion
				ctx = clues.Add(context.Background(), "in-ctx", 2)
				ctx := clues.Add(ctx, "in-ctx", 1)
				err := clues.NewWC(ctx, "err").With("in-err", 1)
				err = clues.StackWC(ctx, err).With("in-err", 2)
```